About/about.html

```diff
@@ -0,0 +1,172 @@
+          </li>
+          <li>
+            <a href="#">English</a>
+          </li>
+          <li>
+            <a href="#">My Page</a>
+          </li>
+          <li>
+            <a href="#">Logout</a>
+          </li>
+        </ul>
+      </nav>
+
+      <nav class="second-title">
+        <div class="ms-5">
+          <a href="../index.html"
+            ><img
+              class="logo-img ms-5"
+              src="../images/festival-logo.jpg"
+              alt="Calabar Carnival logo">
+            </a>
+        </div>
+        <ul class="title-white">
+          <li>
+            <a href="about/about.html" class="nav-link text-muted">About</a>
+          </li>
+          <li>
+            <a href="#" class="nav-link text-muted">Program</a>
+          </li>
+          <li>
+            <a href="#" class="nav-link text-muted">Sponser</a>
+          </li>
+          <li>
+            <a href="#" class="nav-link text-muted">Vendor</a>
+          </li>
+          <li>
+            <a href="#" class="nav-link text-muted">News</a>
+          </li>
+          <li class="pt-1">
+            <a href="#" class="title-red mx-5">CC Campaign</a>
+          </li>
+        </ul>
+      </nav>
+
+      <article>
+        <h2 class="display-2 text-center header ps-3 mt-5">
+          "Hello! Sharing World"
+        </h2>
+        <h1 class="display-1 text-center fw-bold header ps-3 pe-3">
+          The Calabar Carnival is without a doubt the biggest,
+          best and longest tourism event in West Africa.
+        </h1>
+        <p class="intro p-3 bg-white mx-3 border-white mx-5">
+          The Calabar Carnival is without a doubt the biggest, 
+          best and longest tourism event in West Africa.
+        </p>
+        <p class="text-center py-3">
+          Please contact us via email 
+          <br />
+          test@gmail.com
+        </p>
+      </article>
+    </header>
+
+    <section class="text-center">
+      <article>
+        <h3>Calabar Carnival.</h3>
+        <p class="mx-5">
+          It attracts over two million revellers and features participants from over 25 different countries. The Battle of the Bands and Street Parade is even broadcast to a television audience of around fifty million!
+        </p>
+        <img class="img-fluid" src="../images/festival-logo.jpg"
+        alt="calabar carnival logo">
+        <hr class="mt-5" />
+        <h3>See the past festivals</h3>
+        <hr style="width: 10%; height: 3px; color: red; margin-left: 45%" />
+        <p>
+          Take a glimpse of the first along with the second edition of the
+          festival in 2019 & 2021.
+        <div class="flex">
+          <p class="festival-background text-white p-5 fs-5 mx-3">
+            <span class="fw-bold">2022</span> <br />
+            2022 Calabar Carnival 
+          </p>
+          <p class="festival-background text-white p-5 fs-5 mx-3">
+            <span class="fw-bold">2021</span> <br />
+            Calabar Carbival 2020
```

Comment:
-[ ] For this project, you should create your website inspired by the original design provided by Behance. So, your website should look exactly like that design but you could just change the content. This will help you once working in a company to implement what the Design team will provide you like the mockup of the project you're working on.  By the way, I'll suggest you add those two assets with the corresponding text on your about page.
- below is the original screenshot:
![image](https://user-images.githubusercontent.com/71874425/170831735-2be05033-4a12-490b-a7ac-6dc55db7ab33.png)